src/dotnet/commands/dotnet-build/Program.cs

```diff
@@ -15,13 +15,19 @@ namespace Microsoft.DotNet.Tools.Build
 {
     public class BuildCommand
     {
-        public static int Run(string[] args)
+        public static int Run(string[] args) => Run(args, null);
+
+        public static int Run(string[] args, WorkspaceContext workspace)
```

Comment:
Yeah, this isn't even really used right now. It's also totally an RC2 quick-fix. We have some thoughts about refactoring a lot of this code post-RC2 into something cleaner.